src/wagtailmedia/models.py

```diff
@@ -67,12 +67,12 @@ class AbstractMedia(CollectionMember, index.Indexed, models.Model):
     objects = MediaQuerySet.as_manager()
 
     search_fields = CollectionMember.search_fields + [
-        index.SearchField("title", partial_match=True, boost=10),
+        index.AutocompleteField("title", boost=10),
```

Comment:
https://docs.wagtail.org/en/stable/releases/5.0.html#elasticsearch-backend-no-longer-performs-partial-matching-on-search mention `partial_match` should be dropped, and if needed, add an additional `index.AutocompleteField` declaration.

in other words, please keep the `SearchField`s, but without `partial_match`, and add the `AutocompleteField` ones